Gemfile.lock

```diff
@@ -76,47 +77,32 @@ GEM
       descendants_tracker (~> 0.0.1)
     concurrent-ruby (1.1.7)
     crass (1.0.6)
-    cucumber (4.1.0)
```

Comment:
Computer said no to upgrading cucumber?
Investigating this

Reply:
Ok, for some reason I though I was on cucumber-rails 1.4.1, now looking at the commit history I can see that I did not upgrade the project to 1.6.0 neither 1.8.0 as Gemfile.lock had 2.1.0 version (I'm not sure where I took this idea from), that then caused cucumber to be downgraded (obviously)